tests/recipes/test_eleuther_eval.py

```diff
@@ -49,7 +49,7 @@ def test_torchtune_checkpoint_eval_results(self, capsys, monkeypatch, tmpdir):
 
         out = capsys.readouterr().out
 
-        search_results = re.search(r"acc(?:_norm)?\s*\|?\s*([\d.]+)", out.strip())
+        search_results = re.search(r"acc\s*\|\s*\↑\s*\|?\s*(\d+\.\d+)", out.strip())
```

Comment:
This will only work for 0.4.3, right? If so we should also pin the version

Reply:
+1 to this suggestion

BTW, as someone who's regex-challenged, can I request for a quick comment which describes what this line is trying to do? probably less of an issue in the chat-gpt world where we live in, but I think it would be nice to document this

Reply:
Yeah, good point. I think I'd rather modify the RegEx to pick up both options and *not* pin the version. I'd rather explicitly say users can user any of the lm-eval v0.4